edelphi/src/main/java/fi/metatavu/edelphi/auth/AbstractAuthenticationStrategy.java

```diff
@@ -69,8 +71,16 @@ protected User registerExternalUser(String firstName, String lastName, String em
   // TODO If the user has an account activation in progress, should logging in via external sources be denied? 
   
   protected AuthenticationResult processExternalLogin(RequestContext requestContext, String externalId, List<String> emails, String firstName, String lastName) {
+    System.out.println("EXTERNAL LOGIN START");
+    System.out.println("  externalId: " + externalId);
+    System.out.println("  firstName: " + firstName);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)